test/utils.py

```diff
@@ -0,0 +1,71 @@
+from pathlib import Path
+
+
+class DisplayablePath(object):
```

Comment:
This is no longer needed?

Reply:
yeah it is not, I thought I deleted it, thanks for pointing out